ts/components/FetchActivityIndicator.tsx

```diff
@@ -0,0 +1,38 @@
+import { View } from "native-base";
+import * as React from "react";
+import { ActivityIndicator, StyleSheet } from "react-native";
+
+import variables from "../theme/variables";
+
+const styles = StyleSheet.create({
+  activityIndicatorContainer: {
+    position: "absolute",
+    left: 0,
+    right: 0,
+    top: 0,
+    bottom: 0,
+    justifyContent: "center",
+    alignItems: "center",
+    zIndex: 1000
+  }
+});
+
+export type Props = {
+  show: boolean;
```

Comment:
`show` is a verb (more suitable for an action, like `show()`) - here instead is better to have an adjective for a property, e.g. `is_visible: boolean`